app/views/govuk_publishing_components/components/feedback/_problem_form.html.erb

```diff
@@ -39,25 +39,32 @@
       } %>
 
       <%= render "govuk_publishing_components/components/button", {
-        text: t("components.feedback.send")
+        text: t("components.feedback.send"),
+        data_attributes: {
+          ga4: {
+            event_name: "form_submit",
+            type: "feedback",
+            text: t("components.feedback.send"),
```

Comment:
If we use the translated values, performance analysts wouldn't be able to track the clicks properly as there would be 100+ different types of GTM events, containing each languages version of "Send". Therefore I think we need to ensure that the text sent to GTM is always in English.

Reply:
That makes sense. My only concern would be if that text is updated in future - we'd need a way to ensure that this gets updated too. Not sure what that would be.

Reply:
Have updated, thanks. Feels like at least the slightly less bad option of the two choices here.